packages/grafana-ui/src/utils/fieldDisplay.ts

```diff
@@ -63,20 +67,42 @@ export interface FieldDisplay {
   field: FieldConfig;
   display: DisplayValue;
   sparkline?: GraphSeriesValue[][];
+  links?: LinkModel[]; // Links with the values escaped
```

Comment:
Stupid idea maybe, but one potentially huge performance improvement would be to make this lazy, so link interpolation & building the final URL is done on when needed.

Reply:
Changed that in 5887de4

Reply:
I removed it entirely from FieldDisplay... and then only create it on demand when the context menu is opened